wls-auth-service/src/main/java/de/muenchen/oss/wahllokalsystem/authservice/rest/SessionController.java

```diff
@@ -0,0 +1,52 @@
+package de.muenchen.oss.wahllokalsystem.authservice.rest;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+import de.muenchen.oss.wahllokalsystem.authservice.service.SessionService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class SessionController {
+
+    @Autowired
+    SessionService sessionService;
+
+    /**
+     * Lists all sessions which are not expired.
+     *
+     * @return OAuthServerSessions which contains a list of OAuthServerSession (sessionId and
+     *         username)
+     */
+    @PreAuthorize("hasAuthority('ROLE_ADMIN_ADMIN')")
+    @GetMapping(value = "/oauthsessions/")
+    public ResponseEntity<OAuthServerSessions> listActiveSessions() {
+        return ResponseEntity.ok(sessionService.getActiveSessions());
+    }
+
+    /**
+     * Kills sessions which are not expired.
+     *
+     * @param sessionID SessionId of session to kill.
+     * @return HTTP ok if session was killed successfully, HTTP 404 if session to kill was not found
+     *         or session was expired.
+     */
+    @PreAuthorize("hasAuthority('ROLE_ADMIN_ADMIN')")
+    @RequestMapping(value = "/oauthsessions/{sessionID}/invalidate", method = POST)
+    public ResponseEntity<?> killSession(@PathVariable("sessionID") String sessionID) {
```

Comment:
Wenn aus dem Methoden hervorgeht dass eine aktive Session gekillt werden soll ist der Kommentar nicht erforderlich.

```suggestion
    @PreAuthorize("hasAuthority('ROLE_ADMIN_ADMIN')")
    @RequestMapping(value = "/oauthsessions/{sessionID}/invalidate", method = POST)
    public ResponseEntity<?> killSession(@PathVariable("sessionID") String sessionID) {
```